src/components/router/components/ScrollToTop.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+class ScrollToTop extends React.Component {
+    componentDidUpdate(prevProps) {
+        if (this.props.location.pathname !== prevProps.location.pathname) {
+            window.scrollTo(0, 0);
+        }
+    }
+    render() {
+        return this.props.children;
+    }
+}
+
```

Comment:
this changes here is already in develop branch.
remove it

Reply:
Yea i pulled it from develop